src/main/resources/db/migration/V4__create_del_emp_project_link_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE projectDeliveryEmployees (
+    projectId INT NOT NULL,
+    deliveryEmployeeId INT NOT NULL,
+    FOREIGN KEY(projectId) REFERENCES project(projectId),
+    FOREIGN KEY(deliveryEmployeeId) REFERENCES deliveryEmployee(deliveryEmployeeId),
+    PRIMARY KEY(projectId, deliveryEmployeeId)
```

Comment:
is this the right order of primary and foreign key

Reply:
changed